src/library.js

```diff
@@ -2304,6 +2304,9 @@ LibraryManager.library = {
         {{{ makeSetValue('sa', C_STRUCTS.sockaddr_in.sin_family, 'family', 'i16') }}};
         {{{ makeSetValue('sa', C_STRUCTS.sockaddr_in.sin_addr.s_addr, 'addr', 'i32') }}};
         {{{ makeSetValue('sa', C_STRUCTS.sockaddr_in.sin_port, '_htons(port)', 'i16') }}};
+        /* Use makeSetValue instead of memset to avoid adding memset dependency for all users of _write_sockaddr. */
+        assert({{{ C_STRUCTS.sockaddr_in.__size__ - C_STRUCTS.sockaddr_in.sin_zero }}} == 8);
```

Comment:
I wonder if this would work as a "static assert", like `{{{ assert(C_STRUCTS.sockaddr_in.__size__ - C_STRUCTS.sockaddr_in.sin_zero) == 8); }}}`?

If not, then let's wrap this line behind
```
#if ASSERTIONS
assert(...);
#endif
```
block, since in release build in MINIMAL_RUNTIME, the assert() function does not exist.

Reply:
It works if I do this:

`{{{ assert(C_STRUCTS.sockaddr_in.__size__ - C_STRUCTS.sockaddr_in.sin_zero == 8), '' }}}`

I just have no clue what that `, ''` is doing; I just searched for existing instances of `{{{ assert` and `library_syscall.js` does it like this :D Would that be fine?

Otherwise I can ifdef it indeed, many other places do it like that too :)

Reply:
I think the thing in the `{{{ ... }}}` has to resolve to a string.   Perhaps we could make it more flexible?  But it seems OK for now.